python-backend/app/api/mines/incidents/resources/mine_incident_followup_types.py

```diff
@@ -0,0 +1,26 @@
+from flask_restplus import Resource, reqparse, fields, inputs
+from flask import request
+from datetime import datetime
+from werkzeug.exceptions import BadRequest, NotFound, InternalServerError
+
+from app.extensions import api
+from app.api.utils.resources_mixins import UserMixin
+from app.api.utils.access_decorators import requires_role_mine_view, requires_role_mine_create
+
+from ..models.mine_incident_followup_type import MineIncidentFollowupType
+
+mine_incident_followup_type_model = api.model(
```

Comment:
Can you please move these to the mine_api_models file.

Reply:
Moved.